MSAL/src/native_auth/controllers/sign_up/MSALNativeAuthSignUpController.swift

```diff
@@ -243,12 +238,7 @@ final class MSALNativeAuthSignUpController: MSALNativeAuthBaseController, MSALNa
         context: MSIDRequestContext
     ) async -> SignUpStartCodeControllerResponse {
         switch result {
-        case .verificationRequired(let continuationToken, let unverifiedAttributes):
-            MSALLogger.log(
-                level: .info,
-                context: context,
-                format: "verification_required received from signup/start request for attributes: \(unverifiedAttributes)"
-            )
+        case .success(let continuationToken):
```

Comment:
can you add a verbose log here?

Reply:
I've added an `info` log to match the rest of the logs of this class. I've checked the other controllers, and we have:

- SignUpController and ResetPasswordController, that log using mostly `.info`.
- SignInController and CredentialsController, that log using `.verbose`.

I think we should follow the same pattern. Do we create a task for that?

Reply:
Yes please, can you create a PBI under the iOS engineer backlog? Thanks